docs/REST API/rest-api.md

```diff
@@ -3630,6 +3643,79 @@ Retrieves a list of the configuration profiles assigned to a host.
 }
 ```
 
+### Lock host
+
+_Available in Fleet Premium_
+
+Sends a command to lock the specified macOS, Linux, or Windows host. The host is locked once it comes online.
+
+`GET /api/v1/fleet/hosts/:id/lock`
```

Comment:
```suggestion
`POST /api/v1/fleet/hosts/:id/lock`
```

Reply:
@georgekarrv thank you for catching these!! Did some copy/pasting and forgot to change the method, should absolutely be `POST`!